storage/replica_command.go

```diff
@@ -853,6 +853,7 @@ func (r *Replica) PushTxn(batch engine.Engine, ms *engine.MVCCStats, args proto.
 	}
 
 	// Persist the pushed transaction using zero timestamp for inline value.
+	reply.PusheeTxn.Writing = true // required; see comment on the proto
```

Comment:
actually, need to set this before or it doesn't get written. If we error, the reply is moot anyways. Will leave as is.